src/Stripe.net/Services/DateFilterOptions.cs

```diff
@@ -3,21 +3,18 @@
     using System;
     using Newtonsoft.Json;
 
-    public class DateFilter
+    public class DateFilterOptions : INestedOptions
```

Comment:
I think `DateRangeOptions` would be a better name: filtering on an exact date is also a "date filter", but this class now only handles ranges. It would also make the class name consistent with the properties names.